pod/video_search/templates/search/search.html

```diff
@@ -43,160 +43,160 @@ <h3>{% trans "Search results" %}</h3>
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=type.slug:{{ type.key|slugify }}">
+            {{ type.key }}
+          </a>
+          ({{ type.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+    {% if aggregations.tags_name and aggregations.tags_name.buckets %}
+      <dt>{% trans 'Tags' %}</dt>
+      {# Provide only the top 5 type #}
+      {% for tag in aggregations.tags_name.buckets %}
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=tags.slug:{{ tag.key|slugify }}">
+            {{ tag.key }}
+          </a>
+          ({{ tag.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+    {% if aggregations.disciplines_title and aggregations.disciplines_title.buckets %}
+      <dt>{% trans 'Discipline' %}</dt>
+      {# Provide only the top 5 type #}
+      {% for discipline in aggregations.disciplines_title.buckets %}
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=disciplines.slug:{{ discipline.key|slugify }}">
+            {{ discipline.key }}
+          </a>
+          ({{ discipline.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+
+    {% if aggregations.channels_title and aggregations.channels_title.buckets %}
+      <dt>{% trans 'Channel' %}</dt>
+      {# Provide only the top 5 type #}
+      {% for channel in aggregations.channels_title.buckets %}
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=channels.slug:{{ channel.key|slugify }}">
+            {{ channel.key }}
+          </a>
+          ({{ channel.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+
+    {% if aggregations.main_lang and aggregations.main_lang.buckets %}
+      <dt>{% trans 'Main language' %}</dt>
+      {# Provide only the top 5 type #}
+      {% for main_lang in aggregations.main_lang.buckets %}
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=main_lang:{{ main_lang.key }}">
+             {{main_lang.key}}
+          </a>
+          ({{ main_lang.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+
+    {% if aggregations.cursus and result.aggregations.cursus.buckets %}
+      <dt>{% trans 'University course' %}</dt>
+      {# Provide only the top 5 cursus #}
+      {% for cursus in aggregations.cursus.buckets %}
+        <dd>
+          <a href="{{ request.get_full_path }}{% if "?" in request.get_full_path %}&amp;{% else %}?{% endif%}selected_facets=cursus:{{ cursus.key }}">
+             {{cursus.key }}
+          </a>
+          ({{ cursus.doc_count }})
+        </dd>
+      {% endfor %}
+    {% endif %}
+    </dl>
+  </div>
```

Comment:
@Badatos il serait vraiment préférable d'avoir tes regles de formattage dans github action.
@JoshuaBaubry @ptitloup des avis ?

Reply:
très compliqué, je n'ai pas trouvé d'outil automatique qui gère les templates html django :/

Reply:
@Badatos  Tes PR, souvent, réindentent le code donc je suppose que ton editeur doit applique une certaine regle de formattage.
Tu devrais regarder ceci: https://mskelton.medium.com/auto-formatting-code-using-prettier-and-github-actions-ed458f58b7df
![image](https://user-images.githubusercontent.com/36244941/112821894-87e74f00-9087-11eb-9989-1088f00bbc2e.png)

Reply:
Oui c'est ce qu'on discutait sur Rocket. Malheureusement prettier ne gère pas les templates Django (et visiblement ne le fera pas :)
Ceci dit, si déjà on arrivait à éliminer les mixed tab/spaces indentce serait pas mal, et pour ca en effet une github action serait salutaire. On en discute sur le Rocket ? j'ai l'impression qu'on s'éloigne de la PR ;)